src/main/java/baritone/process/MineProcess.java

```diff
@@ -472,7 +493,16 @@ public void mine(int quantity, BlockOptionalMetaLookup filter) {
         this.branchPoint = null;
         this.branchPointRunaway = null;
         this.anticipatedDrops = new HashMap<>();
+        this.validDrops = new HashSet<>();
+        this.blacklistBlocks = new BlockOptionalMetaLookup();
         if (filter != null) {
+            for(BlockOptionalMeta bom : filter.blocks()) {
+                Block block = bom.getBlock();
+                Item item = block.getItemDropped(block.getDefaultState(), new Random(), 0);
+                Item ore = Item.getItemFromBlock(block);
```

Comment:
Isn't there a method in block like block.asItem...

I know there is in newer mc versions, as getItemFromBlock does become deprecated

Reply:
I didn't found the method in this mc version.

Reply:
Ah, alright. I'll leave this comment though for the Baritone maintainers to use asItem for up-porting this